app/UiTPASService/UiTPASServiceEventControllerProvider.php

```diff
@@ -26,44 +29,49 @@ public function connect(Application $app): ControllerCollection
             )
         );
 
-        $app['uitpas.event_card_systems_controller'] = $app->share(
-            function (Application $app) {
-                return new EventCardSystemsController(
-                    $app['uitpas']
-                );
-            }
+        $app[GetCardSystemsFromEventRequestHandler::class] = $app->share(
+            fn (Application $app) => new GetCardSystemsFromEventRequestHandler(
+                $app['uitpas']
+            )
+        );
+
+        $app[SetCardSystemsOnEventRequestHandler::class] = $app->share(
+            fn (Application $app) => new SetCardSystemsOnEventRequestHandler(
+                $app['uitpas']
+            )
+        );
+
+        $app[AddCardSystemToEventRequestHandler::class] = $app->share(
+            fn (Application $app) => new AddCardSystemToEventRequestHandler(
+                $app['uitpas']
+            )
+        );
+
+        $app[DeleteCardSystemFromEventRequestHandler::class] = $app->share(
+            fn (Application $app) => new DeleteCardSystemFromEventRequestHandler(
+                $app['uitpas']
+            )
         );
 
         /** @var ControllerCollection $controllers */
         $controllers = $app['controllers_factory'];
 
-        $controllers->get(
-            '/{eventId}/',
-            GetUiTPASDetailRequestHandler::class
-        )->bind(self::EVENT_DETAIL);
+        $controllers->get('/{eventId}/', GetUiTPASDetailRequestHandler::class)
+            ->bind(self::EVENT_DETAIL);
```

Comment:
As we just discussed on Slack, reminder that we should remove these `bind()` calls and not inject `Symfony\Component\Routing\Generator\UrlGeneratorInterface` in `GetUiTPASDetailRequestHandler`, but instead inject a `CallableIriGenerator` to generate those URLs. Otherwise the URL generation will break when we move these routes with `bind()` calls to the new router, since the Symfony `UrlGeneratorInterface` won't be able to find them in the old router anymore

Reply:
There are also 2 other `bind()` calls in other controller providers. I think those can just be removed, because we don't use `UrlGeneratorInterface` in any other place. I think they are left over from long ago.